ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -225,18 +252,7 @@ export const AccessRenewal = fp.flow(
     getProfile();
   }, []);
 
-
-  // Helpers
-  const getExpirationTimeFor = moduleName => fp.flow(fp.find({moduleName: moduleName}), fp.get('expirationEpochMillis'))(modules);
```

Comment:
moved these 2 up

Reply:
update: removed these 2 since they're trivial now with the new AccessModuleStatus